src/main/java/Duke.java

```diff
@@ -1,10 +1,233 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String LINE_DIVIDER = "\t____________________________________________________________";
```

Comment:
I like the way that you declare a constant here to make the code clean! And good job remembering the format. Most of us would just forget it.